ui.apps/src/main/content/jcr_root/apps/core/fd/components/aemform/v2/aemform/formcontainer.html

```diff
@@ -36,6 +36,7 @@
     <sly data-sly-call="${clientLib.css @ categories=['guide.core.common']}"/>
     <sly data-sly-test.isParentWCMModeDisabled="${wcmmode.disabled}"/>
     <sly data-sly-test.isWCMModePreview="${wcmmode.preview}"/>
-    <sly data-sly-use="${'formcontainer.js' @ wcmmode=isWCMModePreview ? 'preview' : 'disabled', themeOverride=form.themePath, pageFallbackClientlib=guideConstants.THEME_DEFAULT_CLIENTLIB, parentWcmModeDisabled=isParentWCMModeDisabled, afAcceptLang=form.locale}" data-sly-resource="${ @ path=form.formPagePath, wcmmode=isWCMModePreview ? 'preview' : 'disabled'}" />
+    <sly data-sly-use="${'formcontainer.js' @ wcmmode=isWCMModePreview ? 'preview' : 'disabled', themeOverride=form.themePath, pageFallbackClientlib=guideConstants.THEME_DEFAULT_CLIENTLIB, parentWcmModeDisabled=isParentWCMModeDisabled, afAcceptLang=form.locale, embeddedAdaptiveForm='true'}" data-sly-resource="${ @ path=form.formPagePath, wcmmode=isWCMModePreview ? 'preview' : 'disabled'}" />
```

Comment:
Why we have exposed `embeddedAdaptiveForm` in the sightly, should we auto infer ? This does not look readable to me

Reply:
We don't have a mechanism to infer difference between when form container is directly dragged in site page vs form embedded as div. This affects the id generation logic.